dojo_plugin/pages/dojos.py

```diff
@@ -124,7 +145,7 @@ def view_dojo_admin(dojo):
 @dojos.route("/dojo/<dojo>/admin/activity")
 @dojo_route
 def view_dojo_activity(dojo):
-    if not dojo.is_admin():
```

Comment:
Did this slip in by accident? Let's undo this, and then I think we're good to go!